package.json

```diff
@@ -66,9 +66,11 @@
     "eslint": "^6.6.0",
     "eslint-config-prettier": "^6.5.0",
     "eslint-plugin-prettier": "^3.1.1",
+    "eslint-plugin-you-dont-need-lodash-underscore": "^6.8.0",
```

Comment:
Why not.
But I Hope it supports fp :)
(Will try it in m'y branch I havent yet made a PR out of [ https://github.com/vorpaljs-reforged/vorpal/commits/feature/various-improvements]